meson.build

```diff
@@ -54,14 +60,18 @@ if get_option('enable_graphics_debug')
   conf_data.set('WAYFIRE_GRAPHICS_DEBUG', true)
 endif
 
-if get_option('enable_gles32') and meson.get_compiler('cpp').has_header('GLES3/gl32.h')
+if get_option('enable_gles32') and meson.get_compiler('cpp').has_header('GLES3/gl32.h', args: '-I/usr/local/include')
```

Comment:
It seems like this should be from `pkg-config --variable=includedir glesv2` with PKG_CONFIG_PATH correctly set to e.g. /usr/local/lib/pkgconfig

Reply:
Yes, I agree, even this might break on some non-standard distros like NixOS.